src/Item.php

```diff
@@ -151,15 +151,21 @@ public function get_item_tags($namespace, $tag)
     }
 
     /**
-     * Get the base URL value from the parent feed
-     *
-     * Uses `<xml:base>`
+     * Get the base URL value.
+     * Uses `<xml:base>`, or item link, or feed base URL.
      *
      * @param array $element
      * @return string
      */
     public function get_base($element = [])
     {
+        if (!empty($element['xml_base_explicit']) && isset($element['xml_base'])) {
```

Comment:
Same code as https://github.com/simplepie/simplepie/blob/cce6ad227c742e54d1a1170087f3ef13bea06aa5/src/SimplePie.php#L2071-L2080
I am not found of the proposition, because that is exactly what a standard `empty()` is for, and I personally find it easier to read, but I do not mind much 🤓